manifests/base/rbac/role.yaml

```diff
@@ -59,21 +59,22 @@ rules:
 - apiGroups:
   - trainer.kubeflow.org
   resources:
-  - trainjobs
+  - trainingruntimes/finalizers
+  - trainjobs/finalizers
+  - trainjobs/status
   verbs:
-  - create
-  - delete
   - get
-  - list
   - patch
   - update
-  - watch
 - apiGroups:
   - trainer.kubeflow.org
   resources:
-  - trainjobs/finalizers
-  - trainjobs/status
+  - trainjobs
   verbs:
+  - create
+  - delete
```

Comment:
@tenzen-y I created PR to update this RBAC, can you remove it from your change as well please ?
https://github.com/kubeflow/trainer/pull/2626

Reply:
Good call.
I addressed it.